### ISSUE TITLE
How to change Elementor's UI language

### ISSUE THREAD

**Issue by quantissimo** [pojome/elementor#1832]
I set the WordPress language to French but Elementor stays in English.

I can see that the French translation is at 99% here: https://translate.wordpress.org/locale/fr/default/wp-plugins/elementor

How I change Elementor's language?

**Comment by arielk** [comment]
You have to make sure that the language of your Dashboard is set to French in "Settings > General"
If it does not help, open a ticket via support@elementor.com

**Comment by quantissimo** [comment]
I've already set the language in Settings > General

For anybody who needs to know how to resolve this

I resolved the issue by going to Dashboard > Updates and under "Themes" there were "Translations" section where I updated the translations.

Only then did Wordpress download **elementor-fr_FR.po** and **elementor-fr_FR.mo** to the **\wp-content\languages\plugins** folder

**Comment by arielk** [comment]
@quantissimo Absolutely right! I forgot this step :(

**Comment by bouchaoui1** [comment]
hi i have 3 languages on my website , but i want to keep elementor Ui in english wen i customize my pages ,how can i do it ?
thank you

**Comment by Franz333** [comment]
@bouchaoui1 

I am in the same **shit** ! I have been spending ages with the wpml guys to try to sort out this problem.. weeks spent with them to end up hearing that in the end it's not possible.. 

Elementor language is normally based on your wp general language https://elementor.com/help/how-to-change-the-language-of-elementor-and-elementor-pro/

But with WPML, that setting is not accessible anymore as WPML takes it over. I am using wordpress in english but I am building a webiste in Portuguese, French and English. Portuguese being the default language. 

In the beginning I assumed I would be able to get Elementor interface in English (like wordpress) and set the default language of the site to Portuguese (writing all language content in Portuguese and use the WPML to translate that content). And it was actually the case till a certain point where Elementor suddenly switched to Portuguese to display the frontend editor menu.. Actually, some things in Portuguese and things in English.. like a mixed interface. 

My Theme is Astra. Changing to Twenty Twenty makes Elementor back to English ! 

The WPML support said to me recently having Elementor in Portuguese was the normal behavior.

So to sum it up, it's a huge mess. 

And I don't know what makes Elementor used one language or another.. 

I just opened a ticket.. we will see

Good luck to you

**Comment by SteaveRayvon** [comment]
гавно ебаное ваш елементор
